docs/guides/sre/tls/configuring-tls-acme-client.md

```diff
@@ -0,0 +1,122 @@
+---
+slug: /guides/sre/configuring-tls-acme-client
+sidebar_label: 'Configuring automatic TLS provisioning via ACME'
+sidebar_position: 20
+title: 'Configuring ACME client'
+description: 'This guide provides simple and minimal settings to configure ClickHouse to use OpenSSL certificates to validate connections.'
+keywords: ['ACME configuration', 'TLS setup', 'OpenSSL certificates', 'secure connections', 'SRE guide', 'Let`s Encrypt']
+doc_type: 'guide'
+---
+
+import SelfManaged from '@site/docs/_snippets/_self_managed_only_automated.md';
+import configuringSsl01 from '@site/static/images/guides/sre/configuring-ssl_01.png';
+import Image from '@theme/IdealImage';
+import ExperimentalBadge from '@theme/badges/ExperimentalBadge';
+
+# Configuring automatic TLS provisioning via ACME
+
+<ExperimentalBadge/>
+<SelfManaged />
+
+This guide provides describes how to configure ClickHouse to use ACME (RFC8555) protocol.
+With ACME support, ClickHouse can obtain and renew certificates from providers like Let's Encrypt or ZeroSSL automatically.
```

Comment:
I think we should mention (briefly) what the certificate is good for: encrypting client/server and server/server connections.